### DISCUSSION TITLE
What am I missing re the Max Flow test? Numbers don't add up.

### DISCUSSION THREAD

**Discussion by akoolenbourke** [OrcaSlicer/OrcaSlicer#6290] [Q&A]
I was looking to do a max flow rate test, set the option for 23-33 with 0.5mm step. Orca generated the model and I then looked at the previews for both Speed and Flow Rate.

Flow rate says 33 at the max - good. However when I look at the speed, the max is 172mm/s. Given this is with a 0.2mm layer height and 0.42 line width, the formula for flow would give me 0.2 * 0.42 * 172 = ~14mm/s, nowhere near 33. What am I missing here?

![image](https://github.com/user-attachments/assets/b1c8342b-be87-43f7-b139-94919f1023c5)

![image](https://github.com/user-attachments/assets/c95b2526-d7d1-4902-93bd-a42900e656c1)

**Comment by otede** [comment]
It's simple. Throw the "0.2mm layer height and 0.42 line width" out the window ;)
